docs/content/Authentication-&-Authorization.md

```diff
@@ -62,19 +57,105 @@ Authentication is handled outside of Cube.js. A typical use case would be:
    server could include the token in the HTML it serves or provide the token to
    the frontend via an XHR request, which is then stored it in local storage or
    a cookie.
-3. The JavaSript client is initialized using this token, and includes it in
+3. The JavaScript client is initialized using this token, and includes it in
    calls to the Cube.js API.
-
-[link-rest-api]: /rest-api
-[link-cubejs-client-core-ref]: /@cubejs-client-core#cubejs
+4. The token is received by Cube.js, and verified using any available JWKS (if
+   configured)
+5. Once decoded, the token claims are injected into the [security
+   context][ref-sec-ctx].
 
 <!-- prettier-ignore-start -->
 [[info |]]
 | **In development mode, the token is not required for authorization**, but you
-| can still use it to [pass a security context][link-security-context].
+| can still use it to [pass a security context][ref-sec-ctx].
+<!-- prettier-ignore-end -->
+
+## Using JSON Web Key Sets (JWKS)
+
+### Configuration
+
+As mentioned previously, Cube.js supports verifying JWTs using industry-standard
+JWKS. The JWKS can be provided either as a JSON object conforming to [JWK
```

Comment:
```suggestion
JWKS. The JWKS can be provided as a JSON object conforming to [JWK
```

Reply:
@rpaik I forgot to mention both ways of configuring JWKS here.

Reply:
Cool, thanks 😄